cmd/performance-profile-creator/cmd/root.go

```diff
@@ -34,25 +32,23 @@ import (
 
 	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
 	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
-	clientmachineconfigv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
 	v1 "k8s.io/api/core/v1"
-	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
+	// ClusterScopedResources defines the cluster scope resources under
+	// must-gather-dir/quay-io-openshift-kni-performance-addon-operator-must-gather*
 	ClusterScopedResources = "cluster-scoped-resources"
-	CoreNodes              = "core/nodes"
-	MCPools                = "machineconfiguration.openshift.io/machineconfigpools"
-	YAMLSuffix             = ".yaml"
+	// CoreNodes defines substring in the path to obtain node data
+	CoreNodes = "core/nodes"
+	// MCPools defines substring in the path to obtain mcp data
```

Comment:
`defines the subpath, relative to $DONT_REMEMBER_RIGHT_NOW_PLEASE_CHECK, on which we find the machine config pool definitions`